src/pages/Products.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <q-page-container>
+    <q-page class="q-pa-sm q-pt-md">
+      <div class="q-pa-md row items-start q-gutter-lg">
+        <div v-for="product in products" :key="product.image">
+          <product v-bind:image="product.image" />
+        </div>
+      </div>
+    </q-page>
+  </q-page-container>
+</template>
+
+<script>
+import Product from "../components/Product.vue";
+
+export default {
+  data() {
+    return { products: [] };
+  },
+  beforeCreate() {
```

Comment:
@priyankashrma move this to store create a new store called product and place inside action.js file, we should not call the apis inside the .vue files.

Reply:
I have made the required changes, but getProductsAction can't be called. Please check the screenshot.

![image](https://user-images.githubusercontent.com/58728994/122221499-f2619b80-cece-11eb-83c4-dd8688d28200.png)

Reply:
@priyankashrma please call the action inside created hook